docker-for-windows/wsl.md

```diff
@@ -77,6 +77,8 @@ Ensure you have completed the steps described in the Prerequisites section **bef
 
     Optionally, select any additional distributions you would like to enable WSL 2 on.
 
+    Please note that the WSL Integrations bits running in your distro depend on glibc. This make it difficult to run it on musl-based distros such as Alpine Linux. Alpine users can use [this package](https://github.com/sgerrand/alpine-pkg-glibc) to deploy a glibc side by side with musl to run the integration.
```

Comment:
```suggestion
    > **Note**
    >
    > WSL Integration components running in your distro depend on glibc. This can cause issues when running WSL 2 on musl-based distros such as Alpine Linux. Alpine users can use the [alpine-pkg-glibc(https://github.com/sgerrand/alpine-pkg-glibc){:target="_blank" rel="noopener" class="_"}  package to deploy a glibc with musl to run the integration.
```

Reply:
@simonferquel made some suggestions. PTAL and let me know if this sounds correct?

Reply:
nit: double space before `package`, not sure it shows up in the final output (probably not).

Reply:
Fixed. Thanks Mathieu :)